PHPblog.php

```diff
@@ -8,27 +14,23 @@
 	<body>
 		<h1 class='header'> Lindsay's Blog </h1>
 		<h2 class='subtitle'>WORDS FROM A GIRL LEARNING HOW TO CODE </h1>
-		<div class='blogpost'; style=background-color:#bc1a1a;>
-			<h3>Halfway Done with College </h3>
-			<p class='white'>A look back on sophomore year</p>
-			<p class='white'><a href='http://localhost:8888/view_post.php?postId=0'>Read on</a></p>
-		</div>
-
-		<div class='blogpost'; style=background-color:#153466;>
-			<h3>My Summer in St. Louis</h3>
-			<p class='white'> Code, run, sleep.</p>
-			<p class='white'><a href='http://localhost:8888/view_post.php?postId=1'>Read on</a></p>
-		</div>
-
-		<div class='blogpost'; style=background-color:#8fd7e8;>
-			<h3>My opinion on carrots</h3>
-			<p class='white'> The inside scoop on this special veggie</p>
-			<p class='white'><a href='http://localhost:8888/view_post.php?postId=2'>Read on</a></p>
-		</div>
 
+<?php
 
+function PostABlock($blocknumber){
+	$sorter=getSorter();
+	$item = $sorter[$blocknumber];
+		echo "<div class='blogpost'; style='background-color:$item[BlockColor]'>";
```

Comment:
You don't really need separate echo statements here. You could write it like this instead:

```
echo "
   <div class='blogpost'; style='background-color:$item[BlockColor]'>
   <h3>$item[ArticleTitle]</h3>
   <p class='white'>$item[Subtitle]</p>
   <p class='white'><a href=$item[ReadOn]>Read on</a></p>
";
```